### PR TITLE
Uar 1755

### PR DESCRIPTION
**Pull request by axe4000**

### CODE REVIEW

File: src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentKeyPersonnelAction.java [hunk 1]
```diff
@@ -167,7 +167,13 @@ void prepare(ActionForm form, HttpServletRequest request) {
         //need to set this based on route status, permissions...
         pdform.populatePersonEditableFields();
         handleRoleChangeEvents(pdform.getProposalDevelopmentDocument());
-        
+
+        pdform.getQuestionnaireHelper().prepareView();
+        pdform.getS2sQuestionnaireHelper().prepareView();
```
Comment (natonandroid) [inline src/main/java/org/kuali/kra/proposaldevelopment/web/struts/action/ProposalDevelopmentKeyPersonnelAction.java]: no need for S3sQuestionnaireHelper or answers.

File: src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp [hunk 2]
```diff
@@ -155,7 +155,10 @@
     <br/>
 
     <kra-pd:keyPersons/>
-
+    <div id = "workarea">
+        <kra-pd:proposalDevelopmentQuestionnaireAnswersFiltered bean = "${KualiForm.questionnaireHelper}" property = "questionnaireHelper"/>
```
Comment (natonandroid) [inline src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp]: no need to bring all the questions and answers from Questionnaire in this page, just ONE of them, with the id=11018. 
the logic should loop through the questions and answers in the helper and display just this particular one on this page.

File: src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp [hunk 3]
```diff
@@ -155,7 +155,10 @@
...
+        <kra-pd:proposalDevelopmentQuestionnaireAnswersFiltered bean = "${KualiForm.s2sQuestionnaireHelper}" property = "s2sQuestionnaireHelper" forceNonTransparent="${forceTabNonTransparent}"/>
```
Comment (natonandroid) [inline src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp]: No need to display the S2s questions + answers here...

File: src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp (hunk 2 above)
Comment (nataliac06) [reply src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp]: no need to bring all the questions and answers from Questionnaire in this page, just ONE of them, with the id=11018.
the logic should loop through the questions and answers in the helper and display just this particular one on this page.

File: src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp (hunk 3 above)
Comment (nataliac06) [reply src/main/webapp/WEB-INF/jsp/proposaldevelopment/ProposalDevelopmentKeyPersonnel.jsp]: No need to display the S2s questions + answers here...

**Pull request closed without merging**